src/components/DataLayerModal.tsx

```diff
@@ -46,8 +46,9 @@ export default function DataLayerModal({
                     <p className="text-[16px] text-gray-300 text-left mb-6 leading-[24px]">
                         How hot could it get on the hottest day in a given year, under different greenhouse gas emission scenarios?                    </p>
                     <p className="text-[16px] text-gray-300 text-left mb-6 leading-[24px]">
-                            See estimates of annual maxima of daily maximum near-surface air temperature (TASMAX) from NASA Earth Exchange (NEX) Global Daily Downscaled Projections (GDDP) based on simulations of the Coupled Model Intercomparison Project Phase 6 (CMIP6).                    </p>
-
+                        See estimates of annual maxima of daily maximum near-surface air temperature (TASMAX) from NASA Earth Exchange (NEX) Global Daily Downscaled Projections (GDDP) based on simulations of the Coupled Model Intercomparison Project Phase 6 (CMIP6).                    </p>
+                    <p className="text-[16px] text-gray-300 text-left mb-6 leading-[24px]">
+                        The NEX-GDDP-CMIP6 data is calculated on a 0.25°x0.25° latitude and longitude grid and temperature in major cities could be higher than what is displayed within each gridded cell.                        </p>
```

Comment:
```suggestion
                        The NEX-GDDP-CMIP6 data is calculated on a 0.25°x0.25° latitude and longitude grid (representing scales of about 17 miles / 28 km). Temperature in major cities could therefore be higher than what is displayed.                        </p>
```

Reply:
Just a suggestion...

Reply:
Sorry @j08lue I didn't pay attention to this suggestion when applying a hotfix.
FYI @brianmfreitag